remoting/zookeeper/listener.go

```diff
@@ -37,6 +37,10 @@ import (
 	"github.com/apache/dubbo-go/remoting"
 )
 
+var (
+	DefaultTTL = 15 * time.Minute
```

Comment:
```suggestion
	defaultTTL = 15 * time.Minute
```
I think this is a private variable?

Reply:
Yes, fix finished